### DISCUSSION TITLE
How do I import a table?

### DISCUSSION THREAD

**Discussion by SteveRMann** [TriliumNext/Trilium#4059] [General]
I tried importing a markdown:

| Name       | Required Skills | Includes Supervisor | Supports Add-ons | Supports Snapshots | Includes Operating System | Uses Docker | Method               |
|------------|-----------------|---------------------|------------------|--------------------|---------------------------|-------------|----------------------|
| OS         | Minimal (1)     | Y                   | Y                | Y                  | Y                         | Y           | Disk image (2)       |
| Supervised | Linux, Docker   | Y                   | Y                | Y                  | N                         | Y           | Shell script (3)     |
| Container  | Linux, Docker   | N                   | N                | N                  | N                         | Y           | Docker container (4) |
| Core       | Linux           | N                   | N                | N                  | N                         | N           | Python app (5)       |

And I got this in my note:
| Name | Required Skills | Includes Supervisor | Supports Add-ons | Supports Snapshots | Includes Operating System | Uses Docker | Method | |------------|-----------------|---------------------|------------------|--------------------|---------------------------|-------------|----------------------| | OS | Minimal (1) | Y | Y | Y | Y | Y | Disk image (2) | | Supervised | Linux, Docker | Y | Y | Y | N | Y | Shell script (3) | | Container | Linux, Docker | N | N | N | N | Y | Docker container (4) | | Core | Linux | N | N | N | N | N | Python app (5) |

I generate my tables using: https://tablesgenerator.com

What am I doing wrong?

**Comment by sigaloid** [comment]
Are you importing it as a `.md` file and right clicking -> import file? Or are you pasting it in? 

Trilium isn't a Markdown app, it stores in HTML format, but you can _import_ from Markdown explicitly

**Comment by SteveRMann** [comment]
Thanks for the suggestion. I am new with Trilium and still learning.
Initially I was pasting the markup, and after seeing your suggestion I tried importing the table as an .md file.  Both gate the same results.

But, pasting the table in HTML mostly worked.  The table had some code that didn't work, but apparently has no effect:

`.tg {border-collapse:collapse;border-spacing:0;} .tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px; overflow:hidden;padding:10px 5px;word-break:normal;} .tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px; font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;} .tg .tg-0pky{border-color:inherit;text-align:left;vertical-align:top} 
`

Again your assistance is much appreciated.
